styles/components/_modalWindow.mediaQueries.scss

```
@@ -0,0 +1,261 @@
+//TO REMOVE!!!!
+html {
+  overflow: hidden;
+}
+
+@media only screen and (max-width: $breakpoint-xxl-max) {
```

after rwd branch will be merged to develop:
```suggestion
@include xlg {
```